spreadsheets/sort_rank/contours.csv

```diff
@@ -0,0 +1,2 @@
+kind,sort_rank
+*,1500
```

Comment:
This should go between dam and breakwater, so reserve a slot there, and value more like `224` (with breakwater and other values in there getting +1).

https://github.com/tilezen/vector-datasource/blob/a53f1ef0b54c66da391584e25fa750f8aed1a9c9/spreadsheets/sort_rank/landuse.csv

Reply:
Updated according to your comment.